### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by grindhold** [chergert/template-glib#3]
Trying to build on debian stretch.
I got
```
/usr/bin/ld: template_glib_eval-main.o: undefined reference to symbol 'g_file_new_for_commandline_arg'
```
despite autogen.sh running through without any errors.

**Comment by chergert** [comment]
This should have been fixed by commit f0fcd6d941e39797e4e5d99e7f2d395cadb57530.

**Comment by grindhold** [comment]
works :+1:
